### ISSUE TITLE
Please provide the training steps to reproduce the work from scratch, Thanks!

### ISSUE THREAD

**Issue by PingHGao** [eriklindernoren/PyTorch-YOLOv3#453]
In the code, the optimizer is Adam. I trained it for a long time but the loss is around 8 and MAP is around 0.06. This is frustrateing. It is my first time training model from scratch, PLS help! Please someone provide some training details and process.
